presto-main/src/main/java/com/facebook/presto/operator/scalar/ReflectionParametricScalar.java

```diff
@@ -597,7 +654,6 @@ public MethodHandleAndConstructor specialize(Signature boundSignature, BoundVari
             if (nullable) {
                 return Primitives.wrap(clazz);
             }
-            checkArgument(clazz != void.class);
```

Comment:
Did you change the handling for this somewhere? I don't see any changes related to it